### ISSUE TITLE
Submission Format for the challenge

### ISSUE THREAD

**Issue by ilkay87** [sharibox/EAD2019#4]
Couple of questions and answers of interest for the participants.

1-) Is the results submission system online? If not do you plan make it online after the release of test data?
--> You will need to submit it online and all evaluations will be done online. We will set up a leaderboard by 16th February which is also test data release date.

2-) Which format should the test results be submitted (e.g. csv files containing bounding box  locations and class probability scores )?

--> For detection and generalisation, you will need to submit the results as .txt in yolo format (please see our GitHub examples). You will find some test samples there. There are also evaluation codes that we will be using to evaluate. This might help you. If you have any problem exploring them just let us know. 

3-) Will there be some test data completely from a different centre (looking very different from training set)?
--> Yes, it will from a different data centre. This might also look some what different.

4-) For segmentation challenge, should we consider the initial bounding boxes provided in the training data phase-I or generate bounding boxes using segmentation masks?
--> No, you can use the previous bounding boxes as well if you want if you want to do segmentation independent of classification.  

5-) Is it possible to use two different algorithms for segmentations and classification?
--> Yes, absolutely. But, you will need to provide us with bounding boxes for classification as well for those images as '.txt' files (see our answer in 2). 

6-) Is the .txt file we should submit. I checked the Github examples but I do not see any confidence scores there. I guess the submission should include confidence?
--> You have to provide  class_name, confidence, x1, y1, x2, y2 as in the "prediction" folder of our test detection samples. Of course, ground truths are considered to have confidence 1 for all the labels provided.  Just to add, it depends on you what confidence score you want to consider and provide us like threshold at 0.25?

7-) What is the submission format for masks? Should we provide a .tif file similar to ground truth?
--> Yes, the submission format for masks will be a 5 channel ".tif" file.

**Comment by sharib-vision** [comment]
thank you @ilkay87. In case, someone want to get more clear answers, they can just ask here.

**Comment by sharib-vision** [comment]
We have now removed the need for mAP for segmentation. Now, the participants will be evaluated solely on their semantic segmentation. There are other changes too so we refer the participants to see the wiki page: https://github.com/sharibox/EAD2019/wiki/EAD2019-Challenge:-Information-regarding-submission-of-your-results

I am closing this issue now.
